src/store/index.js

```diff
@@ -15,6 +15,9 @@ export default new Vuex.Store({
     online: true,
     autoReloadEnabled: false, // disabled by default
     autoReloadInterval: 30000, // 30 seconds
+    charts: {
+      enabled: true
+    },
```

Comment:
Apologies for misguiding here. Can you help move this inside `modules`, something like `modules.chart_page`, so that it will automatically show in settings page as well. which controls showing/hiding this tab by default ?

ref: https://github.com/linkedin/cruise-control-ui/blob/369d60d620c439307e7baaba2fecd0ab082c4e72/src/store/index.js#L40